src/main/scala/GuessWhoGame/Game.scala

```diff
@@ -1,14 +1,16 @@
 package GuessWhoGame
+
 import scala.util.Random
+import scala.io.StdIn
 
 object Game extends App {
-
+  //stored cases for all different types are allowed - certain restricted numbers of colours etc that could be stored and matched
   val char1: Character = Character("joe", "male", "brown", "blue", wearsGlasses = false, facialHair = false)
   val char2: Character = Character("muhammad", "male", "black", "brown", wearsGlasses = true, facialHair = true)
```

Comment:
I like the idea of making Characters. Enumerations could make this cleaner?

Another way we could make this safer is to think about the boolean values here. Remember the annotations (wearsGlasses etc) are optional, which then means that when a future developer comes along they could get these values confused/round the wrong way. 
One way to overcome this would be to wrap these in their own case classes:
`case class PersonWithGlasses(value: Boolean) `then instead of having wearsGlasses: Boolean inside of Character you could have` wearsGlasses: PersonWithGlasses
`
You could also think about using traits for things like gender, hair colour, eye colour etc. 
```
trait Gender 
case object Male extends Gender
case object Female extends Gender
```
A `Character` can then have the trait `Gender` as the data type for gender parameter` (gender: Gender) `and this will now accept them having any of the case objects that extends this trait. Again, increasing the safety!

Reply:
Attributes have been separated into individual objects and Objects and Values with Case Classes attached for Booleans. CharacterManager Class adapted to generate characters and Character Case Class refactored.